### PR TITLE
🚀  feat: Added Teams Page

### PR DESCRIPTION
**Pull request by Aniket-508**
screencapture-localhost-3000-2022-02-08-18_58_11

### CODE REVIEW

**Comment by Aniket-508** [comment]
@anushbhatia @smaranjitghose Please checkout this PR!

File: src/App.js [hunk 1]
```diff
@@ -2,12 +2,15 @@ import "./App.css";
 import Home from "./components/Home";
 import GlobalStyle from "./globalStyles";
 import Footer from "./components/Footer";
+import Team from "./pages/Team";
+
 function App() {
   return (
     <>
-      <GlobalStyle />
+      {/* <GlobalStyle />
```
Comment (anushbhatia) [inline src/App.js]: create a new page using routing in this.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Could you check the why deployment is failing

1. Might be some warnings left
2. Unused variables present.

**Comment by Aniket-508** [comment]
@anushbhatia The routing actually has to be implemented differently in the navbar which is another issue.

File: src/App.js [hunk 2]
```diff
@@ -2,12 +2,15 @@ import "./App.css";
 import Home from "./components/Home";
```
Comment (anushbhatia) [inline src/App.js]: comment home,global style, footer out and your deployment will work fine

**Pull request merged**